R/add_nodes_from_df_cols.R

```diff
@@ -138,7 +138,7 @@ add_nodes_from_df_cols <- function(graph,
           trimws() %>%
           stringr::str_split(" ") %>%
           purrr::flatten_chr() %>%
-          dplyr::as_tibble() %>%
+          dplyr::as_tibble(.name_repair = "unique") %>%
```

Comment:
I can't really tell what's going on here, but I suspect you'd be able to rewrite this pipeline to eliminate the need to mince around the names of this intermediate object.